src/promptflow-evals/promptflow/evals/synthetic/jailbreak_adversarial_simulator.py

```diff
@@ -134,6 +136,11 @@ async def __call__(
         :keyword concurrent_async_task: The number of asynchronous tasks to run concurrently during the simulation.
             Defaults to 3.
         :paramtype concurrent_async_task: int
+        :keyword language: The language in which the conversation should be generated.
+         example:
+
+         - :py:const:`promptflow.evals.synthetic._constants.SupportedLanguages.English`
```

Comment:
The enum is from a private namespace, is that intentional?

Reply:
changed it.